### DISCUSSION TITLE
Balloon Shooting game in python using pygame.

### DISCUSSION THREAD

**Discussion by Harishik** [community/community#126176] [Repositories]
### Select Topic Area

Show & Tell

### Body

Hi everyone, my name is Harishik,
The following repository has the python code for the intermediate level balloon shooting game I created using Pygame. As a Undergrad student myself, I can understand that many people have difficulties in finding codes for there projects or atleast the basic structure of the code. That is why I would like to share my code with all. Anyone who wants to create a balloon shooting game can just get the repository from here and run it in pycharm or jupyter notebook. I used pycharm but you can use any platform you want, the code will work as long as it supports pygame. Just a reminder that this code won't work in Google Colab as it does not support pygame library.
Thank you, hope it's useful to anyone who needs it. :)
[https://github.com/Harishik/Balloon-Shooter.git](url)

**Comment by queenofcorgis** [comment]
Hi @Harishik , we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
